wasmedge-mysql-nginx/backend/src/main.rs

```diff
@@ -0,0 +1,256 @@
+                    )},
+                ).await?;
+
+            drop(conn);
+            Ok(response_build(serde_json::to_string(&orders)?.as_str()))
+            // Ok(Response::new(Body::from(serde_json::to_string(&orders)?)))
+        }        
+        
+        (&Method::GET, "/delete_order") => {
+            let mut conn = pool.get_conn().await.unwrap();
+
+            let params: HashMap<String, String> = req.uri().query().map(|v| {
+                url::form_urlencoded::parse(v.as_bytes()).into_owned().collect()
+            }).unwrap_or_else(HashMap::new);
+            let order_id = params.get("id");
+
+            "DELETE FROM orders WHERE order_id=:order_id"
+                .with(params! { "order_id" => order_id, })
+                .ignore(&mut conn)
+                .await?;
+
+            drop(conn);
+            Ok(response_build("{\"status\":true}"))
+            // Ok(Response::new(Body::from("{\"status\":true}")))
+        }
+
+        // Return the 404 Not Found for other routes.
+        _ => {
+            let mut not_found = Response::default();
+            *not_found.status_mut() = StatusCode::NOT_FOUND;
+            Ok(not_found)
+        }
+    }
+}
+
+// CORS headers
+fn response_build(body: &str) -> Response<Body> {
+    Response::builder()
+        .header("Access-Control-Allow-Origin", "*")
+        .header("Access-Control-Allow-Methods", "GET, POST, OPTIONS")
+        .header("Access-Control-Allow-Headers", "api,Keep-Alive,User-Agent,Content-Type")
+        .body(Body::from(body.to_owned()))
+        .unwrap()
+}
+
+#[tokio::main(flavor = "current_thread")]
+async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
+    let opts = Opts::from_url(&*get_url()).unwrap();
+    let builder = OptsBuilder::from_opts(opts);
+    // The connection pool will have a min of 5 and max of 10 connections.
+    let constraints = PoolConstraints::new(5, 10).unwrap();
+    let pool_opts = PoolOpts::default().with_constraints(constraints);
+    let pool = Pool::new(builder.pool_opts(pool_opts));
+
+    let addr = SocketAddr::from(([0, 0, 0, 0], 8080));
+    let make_svc = make_service_fn(|_| {
+        let pool = pool.clone();
+        async move {
+            Ok::<_, Infallible>(service_fn(move |req| {
+                let pool = pool.clone();
+                handle_request(req, pool)
+            }))
+        }
+    });
+    let server = Server::bind(&addr).serve(make_svc);
+    if let Err(e) = server.await {
+        eprintln!("server error: {}", e);
+    }
+    Ok(())
+
+    /*
+    let listener = TcpListener::bind(addr).await?;
+    println!("Listening on http://{}", addr);
+    loop {
+        let (stream, _) = listener.accept().await?;
+
+        tokio::task::spawn(async move {
+            if let Err(err) = Http::new().serve_connection(stream, service_fn(handle_request)).await {
+                println!("Error serving connection: {:?}", err);
+            }
+        });
+    }
+    */
+}
```

Comment:
Can you remove this commented code?

Reply:
This is done in 

https://github.com/docker/awesome-compose/pull/309/commits/bbe1a6d1d1e467160e8d7609b6e7e44e3e2d82b1

Thanks.